tests/integration/acp/fixture.go

```diff
@@ -10,7 +10,33 @@
 
 package test_acp
 
+import (
+	"encoding/hex"
+
+	"github.com/decred/dcrd/dcrec/secp256k1/v4"
+	"github.com/sourcenetwork/immutable"
+
+	acpIdentity "github.com/sourcenetwork/defradb/acp/identity"
+)
+
 var (
-	Actor1Identity = "cosmos1zzg43wdrhmmk89z3pmejwete2kkd4a3vn7w969"
-	Actor2Identity = "cosmos1x25hhksxhu86r45hqwk28dd70qzux3262hdrll"
+	Actor1Identity immutable.Option[acpIdentity.Identity]
+	Actor2Identity immutable.Option[acpIdentity.Identity]
 )
+
+func init() {
```

Comment:
question: Why does this need to be in an `init` function?

todo: I would much prefer a util function some what like this:

```go
func toIdentity(privateHex string) immutable.Option[acpIdentity.Identity] {
	privateKeyBytes, err := hex.DecodeString(privateHex)
	if err != nil {
		panic(err)
	}

	return acpIdentity.FromPrivateKey(
		secp256k1.PrivKeyFromBytes(
			privateKeyBytes,
		),
	)
}

var (
	Actor1Identity = toIdentity("028d53f37a19afb9a0dbc5b4be30c65731479ee8cfa0c9bc8f8bf198cc3c075f")
	Actor2Identity = toIdentity("4d092126012ebaf56161716018a71630d99443d9d5217e9d8502bb5c5456f2c5")
)

```

suggestion: Something similar can replace the `loadIdentity` function you have in the other test as well.